shuup/admin/static_src/base/js/drag-n-drop.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * This file is part of Shuup.
+ *
+ * Copyright (c) 2012-2016, Shoop Commerce Ltd. All rights reserved.
+ *
+ * This source code is licensed under the AGPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+window.targetElement = null;
+window.sourceElement = null;
+
+window.updateOrdering = function(e) {
+    var items = ""
+    window.targetElement.find("input[type=checkbox]").each(function(idx, elem) {
+       items += $(elem).attr("name") + "|";
+    });
+    $("#ordering").text(items);
+};
+
+window.removeFromActive = function(e) {
+    e.preventDefault();
+
+    var label = $(this).data("label");
+    var name = $(this).data("name");
+
+    const $source = $("#target-placeholder li");
+    const html = $source.html().replace(/NAME/g, name).replace(/LABEL/g, label);
+    window.sourceElement.append($("<li>").html(html));
+    $(this).closest("li").remove();
+    updateOrdering();
+};
+
+window.addToActive = function(e) {
+    e.preventDefault();
+
+    var label = $(this).data("label");
+    var name = $(this).data("name");
+
+    const $source = $("#source-placeholder li");
+    const html = $source.html().replace(/NAME/g, name).replace(/LABEL/g, label);
+    window.targetElement.append($("<li>").html(html));
+    $(this).closest("li").remove();
+    updateOrdering();
+};
+
+window.activateSortable = function(targetElement, sourceElement) {
+    var $target = $("#" + targetElement);
+    var $source = $("#" + sourceElement);
+    window.targetElement = $target;
+    window.sourceElement = $source;
+
+    var el = document.getElementById(targetElement);
+    var sortable = Sortable.create(el, {
+        handle: ".sorting-handle",
+        onEnd: function (/**Event*/evt) {
+            // evt.oldIndex;  // element's old index within parent
+            // evt.newIndex;  // element's new index within parent
+            updateOrdering();
+        },
+    });
+    updateOrdering();
+    $(document).on("click", ".btn-remove-sortable", removeFromActive);
```

Comment:
This entire file introduces a whole bunch of global functions that aren't used anywhere other than view settings. This should be put into the extra_js of the template. At the very least, wrap this in a IIFE.